src/spanningtrees/kruskal.jl

```diff
@@ -6,27 +6,40 @@ end
 isless(e1::KruskalHeapEntry, e2::KruskalHeapEntry) = e1.dist < e2.dist
 
 """
-    quick_find!(vs, p, q)
+    find_set!(vs, q)
 
 Perform [Quick-Find algorithm](https://en.wikipedia.org/wiki/Disjoint-set_data_structure)
-on a given pair of vertices `p`and `q`, and make a connection between them in the vector `vs`.
+on a given vertice `q`, finds the component id it belongs to.
```

Comment:
"vertex" is the singular. Suggest
"Perform ... on vertex `q` and return the component id in `vs` to which it belongs."